presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergMetadataListing.java

```diff
@@ -120,7 +121,7 @@ public void testTableDescribing()
         assertQuery("DESCRIBE iceberg.test_schema.iceberg_table1", "VALUES ('_string', 'varchar', '', ''), ('_integer', 'integer', '', '')");
     }
 
-    @Test
+    @Test (timeOut = TEST_TIMEOUT)
```

Comment:
Looks good.  Just add a comment explaining that the `iceberg.hive.table-refresh.max-retry-time` property is important, and if the test starts to fail, to look at your code change in `HiveTableOperations`.